File: src/main/kotlin/net/atos/zac/zoeken/model/FilterParameters.kt
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
 * SPDX-License-Identifier: EUPL-1.2+
 */
package net.atos.zac.zoeken.model

import jakarta.json.bind.annotation.JsonbProperty
import nl.lifely.zac.util.AllOpen
import nl.lifely.zac.util.NoArgConstructor

@AllOpen
@NoArgConstructor
data class FilterParameters(
    @field:JsonbProperty("waarden")
    var values: List<String>,

    var inverse: Boolean
)

File: src/main/app/src/app/zoeken/model/filter-parameters.ts
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

export class FilterParameters {
  values: string[];
  inverse: string;

  constructor(values: string[], inverse: boolean) {
    this.values = values;
    this.inverse = String(inverse);
  }
}
